recipe/meta.yaml

```diff
@@ -15,20 +16,20 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
```

Comment:
upstream still says 3.8... https://github.com/CadQuery/cadquery/blob/master/setup.py#L51

Reply:
again, what is the motivation for not using he upstream pin?